doc/user_guide/configuration/naming-styles.rst

```diff
@@ -0,0 +1,192 @@
+.. -*- coding: utf-8 -*-
```

Comment:
Is this file auto-generated, or should I review it ?

Reply:
It's the old page that I moved to a new file (see the diff). I changed nothing but the title